File: docs-old/notintoc/codesnippet/CSharp/menucommands-vs.-olemenucommands_10.cs
                // Create the command for the menu item.
                CommandID menuCommandID = new CommandID(GuidList.guidMenuTextCmdSet, (int)PkgCmdIDList.cmdidMyCommand);
                OleMenuCommand menuItem = new OleMenuCommand(MenuItemCallback, menuCommandID );
                menuItem.BeforeQueryStatus +=
                    new EventHandler(OnBeforeQueryStatus);
                mcs.AddCommand(menuItem);

File: docs-old-1102/misc/codesnippet/VisualBasic/menucommands-vs-olemenucommands_10.vb
            Dim menuCommandID As CommandID = New CommandID(GuidList.guidMenuTextCmdSet, CInt(PkgCmdIDList.cmdidMyTextCommand))
            Dim menuItem As OleMenuCommand = New OleMenuCommand(New EventHandler(AddressOf MenuItemCallback), menuCommandID)
            AddHandler menuItem.BeforeQueryStatus, AddressOf OnBeforeQueryStatus
            mcs.AddCommand(menuItem)